x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json

```diff
@@ -501,7 +526,7 @@
             "default_timefield": "@timestamp",
             "id": "61ca57f0-469d-11e7-af02-69e470af7417",
             "index_pattern": "",
-            "interval": "5m",
+            "interval": "\u003e=5m",
```

Comment:
Is there a reason why some intervals are `>=15m` and others are `>=5m`?

Reply:
The visualizations with `>=15m` interval are showing the `Top N` data points. Because these metrics have an ingest delay of 10-minute and sample period of 5-minute. That's why the interval is hardcoded for `>=15m`. 

For other visualizations showing `Time Series` data, they have 2-minute ingest delays. With storage metricset recommends to use `5m` for collection period, I think `>=5m` visualization interval will work 😄

Reply:
That makes sense. Thanks for explaining.